charts/akeyless-api-gateway/values.yaml

```diff
@@ -33,7 +33,10 @@ deployment:
   nodeSelector:
     #     iam.gke.io/gke-metadata-server-enabled: "true"
 
-
+  securityContext:
+    enabled: false
+    fsGroup: 1001
+    runAsUser: 1001
```

Comment:
Same

Reply:
Just an example . I will put there 0 for root